src/MagnumPlugins/DdsImporter/Test/DdsImporterTest.cpp

```diff
@@ -241,6 +237,23 @@ void DdsImporterTest::dxt5() {
             TestSuite::Compare::Container);
 }
 
+void DdsImporterTest::dxt10rgba() {
+    DdsImporter importer;
+
+    CORRADE_VERIFY(importer.openFile(Utility::Directory::join(DDSIMPORTER_TEST_DIR "/Dxt10TestFiles", "2D_R32G32_FLOAT.DDS")));
+    CORRADE_VERIFY(importer.openFile(Utility::Directory::join(DDSIMPORTER_TEST_DIR "/Dxt10TestFiles", "2D_R8G8B8A8_UINT.DDS")));
```

Comment:
btw., the uppercase extension would make this broken on Linux -- the files have lowercase as far as I see ;)